### PR TITLE
No Translation Requests - Issues & PR Template

### PR DESCRIPTION
**Pull request by scannillo**
### Summary
-Added PR template with note that we cannot accept public translation requests.
-Updated the issue template to note the same.
-Updated the CONTRIBUTING section off the README

### Checklist

- [ ] Added a changelog entry (Did not feel this should be in Changelog)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -7,3 +7,5 @@
 ### Issue description
 
 <!-- To help us quickly reproduce your issue, include as many details as possible, such as logs, steps to reproduce, and so on.  If the issue reports a new feature, follow the [user story](https://en.wikipedia.org/wiki/User_story) format to clearly describe the use case. -->
+
+<!-- NOTE: Please do not open an issue for translation requests. We support the same languages that are supported by PayPal, and use PayPal provided translations. -->
```
Comment (crookedneighbor) [inline .github/ISSUE_TEMPLATE.md]: How about:

```
<!-- NOTE: Please do not open an issue for translation requests for new languages. We support the same languages that are supported by PayPal, and have a dedicated localization team to provide the translations.

If there is an error in a specific translation, you may open an issue here and we will escalate it to the localization team.
-->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ### Summary
 
+<!-- NOTE: We cannot accept language translation PRs. We support the same languages that are supported by PayPal, and use PayPal provided translations. -->
```
Comment (crookedneighbor) [inline .github/PULL_REQUEST_TEMPLATE.md]: How about:

```
<!-- NOTE: We cannot accept language translation PRs. We support the same languages that are supported by PayPal, and have a dedicated localization team to provide the translations. 

If there is an error in a specific translation, you may open an issue and we will escalate it to the localization team.
-->
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ Thanks for considering contributing to this project. Ways you can help:
 * [Add an issue](https://github.com/braintree/braintree-web-drop-in/issues)
 * [Contact us](README.md#feedback) with feedback
 
+__Note on Translations:__ We cannot accept language translation requests. We support the same [languages that are supported by PayPal](https://developer.paypal.com/docs/api/reference/locale-codes/), and use PayPal provided translations.
```
Comment (crookedneighbor) [inline CONTRIBUTING.md]: `and have a dedicated localization team to provide the translations.`

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (scannillo) [reply .github/ISSUE_TEMPLATE.md]: Changes applied!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (scannillo) [reply .github/PULL_REQUEST_TEMPLATE.md]: Changes applied

File: CONTRIBUTING.md (hunk 3 above)
Comment (scannillo) [reply CONTRIBUTING.md]: Changes applied

Comment (crookedneighbor) [reply CONTRIBUTING.md]: This one doesn't look like it got updated.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ Thanks for considering contributing to this project. Ways you can help:
 * [Add an issue](https://github.com/braintree/braintree-web-drop-in/issues)
 * [Contact us](README.md#feedback) with feedback
 
+__Note on Translations:__ We cannot accept language translation requests. We support the same [languages that are supported by PayPal](https://developer.paypal.com/docs/api/reference/locale-codes/), use PayPal provided translations, and have a dedicated localization team to provide the translations.
```
Comment (crookedneighbor) [inline CONTRIBUTING.md]: Sorry, I wasn't clear. I meant we should replace `, use PayPal provided translations` with `and have a dedicated localization team to provide the translations`

Comment (scannillo) [reply CONTRIBUTING.md]: 👍

**Pull request merged**